### PR TITLE
Update configure-email-forwarding.md

### PR DESCRIPTION
**Pull request by msbemba**
Added an important note about controlling automatic forwarding to external restrictions

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3490

File: microsoft-365/admin/email/configure-email-forwarding.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ description: "Set up email forwarding to one or more email accounts using Office
   
 As the admin of an organization, you might have company requirements to set up email forwarding for a user's mailbox. Email forwarding lets you forward email messages sent to a user's mailbox to another user's mailbox inside or outside of your organization.
 
+> [!Important]
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/configure-email-forwarding.md]:
```suggestion
> [!IMPORTANT]
```

File: microsoft-365/admin/email/configure-email-forwarding.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ description: "Set up email forwarding to one or more email accounts using Office
...
+> You can use outbound spam filter policies to control automatic forwarding to external recipients. For more information, View [Control automatic external email forwarding in Microsoft 365](https://docs.microsoft.com/microsoft-365/security/office-365-security/external-email-forwarding?view=o365-worldwide#how-the-outbound-spam-filter-policy-settings-work-with-other-automatic-email-forwarding-controls). If you would
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/configure-email-forwarding.md]:
```suggestion
> You can use outbound spam filter policies to control automatic forwarding to external recipients. For more information, see [Control automatic external email forwarding in Microsoft 365](https://docs.microsoft.com/microsoft-365/security/office-365-security/external-email-forwarding?view=o365-worldwide#how-the-outbound-spam-filter-policy-settings-work-with-other-automatic-email-forwarding-controls).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks.

**Pull request merged**